packages/firebase_ai/firebase_ai/example/lib/pages/bidi_page.dart

```diff
@@ -65,13 +63,20 @@ class _BidiPageState extends State<BidiPage> {
       ],
     );
 
+    // ignore: deprecated_member_use
     _liveModel = FirebaseAI.vertexAI().liveGenerativeModel(
       model: 'gemini-2.0-flash-exp',
       liveGenerationConfig: config,
       tools: [
         Tool.functionDeclarations([lightControlTool]),
       ],
     );
+    initAudioOutput();
+  }
+
+  Future<void> initAudioOutput() async {
```

Comment:
Can this be private? Consider renaming `_initAudio` since it handles both input and output